.vscode/launch.json

```diff
@@ -14,7 +14,7 @@
            "cwd": "${workspaceRoot}/HoneyDo.Web",
            "stopAtEntry": false,
            "launchBrowser": {
-               "enabled": true,
+               "enabled": false,
```

Comment:
This is a personal preference, but I find it very annoying that it opens the browser every time I launch or re-launch the app.